nntrainer/tensor/tensor_base.cpp

```diff
@@ -176,14 +176,14 @@ void TensorBase::getSharedDataTensor(const TensorDim dim_, size_t offset,
   createSharedDataTensor(this, ret, offset);
 }
 
-TensorBase::BroadcastInfoV2
-TensorBase::computeBroadcastInfo(const TensorV2 &m) const {
+TensorBase::BroadcastInfo
+TensorBase::computeBroadcastInfo(const Tensor &m) const {
   if (m.size() > this->size())
     throw exception::not_supported("broadcasting *this is not supported");
 
   const TensorDim m_dim = m.getDim();
 
-  BroadcastInfoV2 e;
+  BroadcastInfo e;
```

Comment:
Is there a reason why BroadcastInfo's variable name is e?

Reply:
no particular reason! reused the previous name.